### DISCUSSION TITLE
Is there a support for Tiny Plots and punch card like scatter plot in Ant charts?

### DISCUSSION THREAD

**Discussion by vaidhykumar** [ant-design-blazor/ant-design-blazor#3710] [Q&A]
I am wondering if there is a support for Tiny Plots and punch card like scatter plot in Ant charts?  I looked at the blazor examples and did not find any.

![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/31709362/25df59eb-d017-41db-93d3-cf519a3b7cbd)
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/31709362/91bff0d7-4095-4228-bf73-44a151ecbb7d)

**Comment by ElderJames** [comment]
Thanks for contacting us @vaidhykumar . Our charts is warpper the G2plot.

**Reply by vaidhykumar** [reply]
Thanks. Any guidance on how I can change  y axis labels to days of week ?
